.github/workflows/reviews.yml

```diff
@@ -0,0 +1,99 @@
+name: "Request reviews"
+on:
+  pull_request_target:
+    types: [labeled]
+  workflow_run:
+    workflows: [Pull Request Labeler]
+    types: [completed]
+jobs:
+  request-reviewer:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/github-script@v5
+        with:
+          script: |
+            // The number of the pull request that triggered this run. If label
+            // was added manually by a person the number will be stored in current
+            // context, otherwise the number will be stored in the payload.
+            const pr_number = context.issue.number || context.payload.workflow_run.pull_requests[0].number
```

Comment:
This seems to fail.

Reply:
Ugh, probably missing some permission. Maybe "content"? I'll need.to research. @jamessan: thoughts?

Reply:
Failure after labeler finishes: <https://github.com/neovim/neovim/runs/5151031859?check_suite_focus=true>

Reply:
Ye. It can't find the context of the previous workflow for some reason. I'll fix it after I wake up properly.

Reply:
https://github.com/neovim/neovim/pull/17370